docker-compose.yml

```diff
@@ -1,9 +1,14 @@
-version: '3.8'
-
```

Comment:
explanation: version is deprecated https://docs.docker.com/reference/compose-file/version-and-name/#version-top-level-element-obsolete